imap_processing/glows/l2/glows_l2_data.py

```diff
@@ -0,0 +1,123 @@
+"""Module containing the class definition for the HistogramL2 class."""
+
+from dataclasses import InitVar, dataclass, field
+
+import numpy as np
+
+
+@dataclass
+class HistogramL2:
+    """
+    Dataclass describing Histogram L2 data variables and methods.
+
+    This class collects multiple HistogramL1B classes into one L2 per observational day.
+
+    flight_software_version
+    number_of_good_l1b_inputs: int
+        number of good-time Level-1B times used for generation of Level-2 data
+    total_l1b_inputs: int
+      number of all Level-1B times for observational day
+    identifier
+        unique Level-2 histogram identifier
+    start_time
+        UTC start time of a given observational day
+    end_time
+        UTC end time of a given observational day
+    daily_lightcurve
+        arrays for observational-day-accumulated lightcurve
+    filter_temperature_average
+        observational-day-averaged filter temperature [Celsius deg]
+    filter_temperature_variance
```

Comment:
I am surprised mypy didn't complain about these doc strings. Can you add datatype for remaining variables?